media-video/mplayer/mplayer-1.4.ebuild

```diff
@@ -298,6 +300,13 @@ src_configure() {
 	use samba || myconf+=" --disable-smb"
 	use lirc  || myconf+=" --disable-lirc --disable-lircc --disable-apple-ir"
 
+	# enable openssl support for https
+	# the binary cannot be redistributed if set
+	# by default gnutls will be enabled
+	if use ssl; then
+		myconf+=" --disable-gnutls --enable-openssl-nondistributable"
+	fi
```

Comment:
Likewise:
```
if use ssl ; then
        myconf+=" $(use_enable gnutls) $(use_enable !gnutls openssl-nondistributable)"
else
        myconf+=" --disable-gnutls --disable-openssl-nondistributable"
fi
```
By the way, you could earn bonus points by converting `myconf` into a bash array. :wink:

Reply:
The configure script does not have `--disable-openssl-nondistributable` configure flag looks like it will have to be omitted.

Reply:
Looked into this further and there are only two configure flags available `--disable-gnutls` and `--enable-openssl-nondistributable` there are not configure flags for `--disable-openssl-nondistributable` and `--enable-guntls`. Thus looking like we can't rely on `use_enable` this case.

The side-effect of this is `QA Notice: command not found:` at the end of the build.

Reply:
I will await feedback on this one before I resolve it

Reply:
```
if use ssl ; then
        myconf+=" $(usex !gnutls '--disable-gnutls --enable-openssl-nondistributable' '')
else
        myconf+=" --disable-gnutls"
fi
```